src/app/view/IAP/ClouderyOffer.tsx

```diff
@@ -35,18 +52,33 @@ const ClouderyOfferWithIAPContext = (): JSX.Element | null => {
 export const ClouderyOffer = withIAPContext(ClouderyOfferWithIAPContext)
 
 interface WebViewWithLoadingOverlayProps {
+  hidePopup: () => void
   popupUrl: string
   interceptNavigation: (request: WebViewNavigation) => boolean
 }
 
 const WebViewWithLoadingOverlay = ({
+  hidePopup,
   popupUrl,
   interceptNavigation
 }: WebViewWithLoadingOverlayProps): JSX.Element => {
   const [loading, setLoading] = useState(true)
+  const { t } = useI18n()
 
   return (
     <View style={styles.dialog}>
+      <View style={styles.headerStyle}>
+        <TouchableOpacity
+          activeOpacity={0.5}
+          onPress={hidePopup}
+          style={styles.headerTouchableStyle}
+        >
+          <BackTo color={colors.primaryColor} width={16} height={16} />
+          <Text style={styles.headerTextStyle}>
+            {t('screens.clouderyOffer.backButton')}
+          </Text>
+        </TouchableOpacity>
+      </View>
```

Comment:
nit: Could be taken out as a separate component to avoid having lot of style stuff in ClouderyOffer.tsx